sdk/samples/iot/iot_sample_common.h

```diff
@@ -149,13 +150,13 @@ typedef enum
 {
   PAHO_IOT_HUB_C2D_SAMPLE,
   PAHO_IOT_HUB_METHODS_SAMPLE,
-  PAHO_IOT_HUB_PNP_COMPONENT_SAMPLE,
-  PAHO_IOT_HUB_PNP_SAMPLE,
   PAHO_IOT_HUB_SAS_TELEMETRY_SAMPLE,
   PAHO_IOT_HUB_TELEMETRY_SAMPLE,
   PAHO_IOT_HUB_TWIN_SAMPLE,
   PAHO_IOT_PROVISIONING_SAMPLE,
-  PAHO_IOT_PROVISIONING_SAS_SAMPLE
+  PAHO_IOT_PROVISIONING_SAS_SAMPLE,
+  PAHO_IOT_PNP_SAMPLE,
+  PAHO_IOT_PNP_WITH_PROVISIONING_SAMPLE
```

Comment:
PnP before Provisioning -- keeps order same here, in readme, and in cmake.
```suggestion
  PAHO_IOT_PNP_SAMPLE,
  PAHO_IOT_PNP_WITH_PROVISIONING_SAMPLE,
  PAHO_IOT_PROVISIONING_SAMPLE,
  PAHO_IOT_PROVISIONING_SAS_SAMPLE
```

Reply:
fixed.